app/services/database_service.rb

```diff
@@ -0,0 +1,34 @@
+class DatabaseService
+    def self.get_url(url, params = nil)
+        response = conn.get(url)
+        JSON.parse(response.body, symbolize_names: true)
+    end
+    
+    def self.conn
+        Faraday.new(url: "http://localhost:3001")
+    end
+    
+    def self.recipe(meal)
+        # get_url("/api/v1/recipe", {meal: meal})
+
+        # temporary, remove when BE endpoints are finished
+        response = File.read('spec/fixtures/recipe_fixture.json')
+        JSON.parse(response, symbolize_names: true)
+    end
+    
+    def self.groups
+        # get_url("/api/v1/groups")
+        
+        # temporary, remove when BE endpoints are finished
+        response = File.read('spec/fixtures/groups_fixture.json')
```

Comment:
This backend endpoint is available; we should coordinate a test run to see if we're able to confirm in development